packages/lesswrong/styles/_sequences.scss

```diff
@@ -24,9 +7,6 @@
     position: absolute;
     right: 0px;
     cursor: pointer;
-    .remove-icon {
```

Comment:
A bit surprised you removed this – it's still listed in the file (Within this commit, at least)

Reply:
Had a long conversation about this. It was fixed in a later PR (oops).